vuepress/docs/next/tutorials/compose/content-tutorial.md

```diff
@@ -1,108 +1,89 @@
+This tutorial describes how to create, edit and publish content in an application with the Entando Web Content Management System from within the App Builder.
 
 ## Creating Content
 
-To see the list of Content in the WCMS:
+1. Log into your App Builder.
 
--   Go to `Content → Management`. All existing 
-    content is displayed in a table list.
+2. From the left sidebar, go to `Content` → `Management`. All existing 
+    content is displayed in a table.
 
-![image](./img/Content1.png)
+![image](./img/content-management.png)
 
-To create a new Content item and add it to the list
+3. To create a new `Content` item, click the `Add` button on the right side of page. A drop-down menu will open from where you select the type. New types can be added to the list in the [Content Type](./content-types-tutorial.md) section.
 
--   Select the **Add Content** button. A drop-down list will let you
-    choose which Content Type the content will be based on.
+![image](./img/content-add.png)
 
-![image](./img/Content2.png)
+4. You will be asked to provide the following information.
+    * `Info` 
 
-You will be asked to provide the following information:
+         * `Content Type`: This is a pre-defined list of the `Types` available.
 
--   **Info**
+         * `Description`: The specific content's name or description. This field is required.
 
-    -   **Content Type**. This information is pre-defined, and lists the
-        content type for the content.
+    *   `Groups`
 
-    -   **Description**. The specific Content's name or description. This field
-        is mandatory.
+         * `Owner Group`: Choose the available user group to manage the content from the drop-down list. Click the `+` button to select. This field is required and predefined in the [Entando Identity Management System](../../docs/consume/identity-management.md).
 
--   **Groups**
+         * `View Only Groups`: Add groups to provide viewing access to the content.
 
-    -   **Owner Group.** Use the drop-down list to choose the
-        available user group that can manage the content.
-        This field is mandatory and may be predefined.
 
-    -   **Join Group.** Use the drop-down list to choose the
-        available user group to share the content with and select +. The group
-        selected will only be able to share and can not
-        administrate this content. This field is not mandatory.
+    *   `Categories`: Use `+` under `Join` to add to the applicable category tree for use in searching and sorting.  This field is not mandatory.
 
-    -   **Categories.** Select the category(s) and select + to add a classification to the 
-        content to use for searching and sorting.  This field is not mandatory.
+    *  `Content Attributes`: Define each required `Attributes` field and any optional fields for the `Content Type` of this item. Only the `Date` and `Title` are required.
```

Comment:
this is awkward/confusing. maybe rephrase to: "It is mandatory to define the `Date` and `Title` of the `Content Type`. You may also define any optional `Attributes` fields."